src/InterestPayout.Common/Persistence/ReadModels/AssetInterests/AssetInterestRepository.cs

```diff
@@ -29,6 +29,19 @@ public async Task Update(AssetInterest assetInterest)
             await _dbContext.SaveChangesAsync();
         }
 
+        public async Task<int> DeleteByAssetIds(IReadOnlyCollection<string> assetIds)
+        {
+            var entities = await _dbContext.AssetInterests
```

Comment:
Let's use Z.EntityFramework.Extensions bulk delete instead

Reply:
Unfortunately, both ways that I know of (`BulkDeleteAsync` and `DeleteFromQueryAsync`) result in the same error: `ERROR_005: The monthly trial period is expired`.

Reply:
Check this https://github.com/swisschain/Sirius.Executor/blob/4e2fdbb8b99f8faf9f39d30d7baa7abc6527e96e/src/Executor.Common/Persistence/Entities/AddressLocks/AddressLocksRepository.cs#L80

Reply:
`+`